rsopt/pkcli/pack.py

```diff
@@ -2,9 +2,11 @@
 import sirepo.sim_data  # Can't run import sirepo and call sirepo.sim_data. ??
```

Comment:
It would be great if some of these general utility functions were moved out of the pkcli command and in to somewhere they are more readily available to other methods -- if they're needed elsewhere as well

Reply:
Is this referring to pack or something with sirepo?
The pkcli functions are as available as anything else in the package.

Reply:
`from rsopt.pkcli import _get_lattice_data` seems a bit stranger to me than `from rsopt.util import function` but that's purely to separate based on structure. That would be a different PR anyway

Reply:
That's something we could consider in the future. Currently all the utility functions in the pkcli modules are used only by that module so it seemed natural to keep them tightly coupled and not clutter up util.